src/components/ui/pagination.tsx

```
@@ -0,0 +1,127 @@
+  ChevronLeftIcon,
+  ChevronRightIcon,
+  MoreHorizontalIcon,
+} from "lucide-react"
+
+import { cn } from "@/lib/utils"
+import { Button, buttonVariants } from "@/components/ui/button"
+
+function Pagination({ className, ...props }: React.ComponentProps<"nav">) {
+  return (
+    <nav
+      role="navigation"
```

Biome (1.9.4)

[error] Using the role attribute 'navigation' on the 'nav' element is redundant, because it is implied by the semantic 'nav' element.

Unsafe fix: Remove the role attribute.


(lint/a11y/noRedundantRoles)